independent-projects/qute/generator/src/main/java/io/quarkus/qute/generator/ValueResolverGenerator.java

```diff
@@ -130,7 +140,7 @@ public void generate(ClassInfo clazz) {
 
         valueResolver.close();
 
-        if (!clazz.superName().equals(OBJECT)) {
+        if (!ignoreSubclasses && !clazz.superName().equals(OBJECT)) {
```

Comment:
This suspiciously looks like it should be `ignoreSuperclasses` no?

Reply:
Eh, yes it's a wrong name! Thanks for spotting this. I'll send a follow-up PR shortly.

Reply:
:)